client-course-schedulizer/src/components/reuseables/AddSectionPopover/AddSectionPopover.tsx

```diff
@@ -150,6 +216,20 @@ export const AddSectionPopover = ({ values }: AddSectionPopover) => {
           }}
           value={(values?.meeting.duration || "").toString()}
         />
+        <GridItemTextField
+          label="Year"
+          register={register}
+          textFieldProps={{ name: "year" }}
+          value={(values?.section.year || "").toString()}
+        />
```

Comment:
In the long term, I think Year will be set for the whole schedule, but this is definitely fine for now